box.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+
+class Box:
+    def __init__(self, box_dim):
+        self.box_dim = box_dim
+        
+    def wrap(self, coordinates):
+        """Wraps the coordinates within the box dimensions
+
+        Parameters
+        ----------
+        box_dim : np.array
+            Dimensions of the box in reduced units.
+        coordinates : np.array
+            Array of the atomic coordinates.
+
+        Returns
+        -------
+        coordinates : np.array
+            Arrays of the wrapped atomic coordinates.
+        """
+        if len(coordinates.shape) == 1:
+            coordinates -= self.box_dim * np.round(coordinates / self.box_dim)
+        else:
+            coordinates -= self.box_dim * np.round(coordinates / self.box_dim[np.newaxis, :])
+        return coordinates
+        
+    def minimum_image_distance(self,coord_i, coord_j):
+        """Calculate the minimum distance between two atoms.
+        
+        Parameters
+        ----------
+        coord_i, coord_j : np.array
```

Comment:
These should be on different lines.

Reply:
Alright. I forgot to change that coord_i and coord_j are floats.

Reply:
Well they are `numpy.ndarray(np.float)`s.